src/firmware/app/primitives/direct_control_primitive.c

```diff
@@ -100,8 +99,8 @@ void app_direct_control_primitive_start(TbotsProto_DirectControlPrimitive prim_m
         }
         default:
         {
-            // Do nothing
-            TLOG_WARNING("Chick command is not a valid type");
+            // If we get here, then we currently don't have any chick command to execute
+            break;
```

Comment:
is this now a valid case?

Reply:
yeah it is, robot diagnostics sends an empty message when we don't want kick,chip,autokick or autochip